scene/gui/text_edit.cpp

```diff
@@ -1664,17 +1664,22 @@ void TextEdit::backspace_at_cursor() {
 	cursor_set_column(prev_column);
 }
 
-void TextEdit::indent_selection_right() {
+void TextEdit::indent_right() {
 
-	if (!is_selection_active()) {
-		return;
-	}
+	int start_line;
+	int end_line;
 	begin_complex_operation();
-	int start_line = get_selection_from_line();
-	int end_line = get_selection_to_line();
+	
```

Comment:
If I'm reading/interpreting the Travis CI output correctly, it thinks you've violated a code formatting rule.  I believe it's asking you to delete two tab characters.